src/rollup/types.d.ts

```diff
@@ -354,12 +344,17 @@ export interface RenderedModule {
 	originalLength: number;
 }
 
-export interface OutputChunk {
+export interface RenderChunk {
```

Comment:
Maybe the name should actually be `RenderedChunk` as it is not a chunk for rendering but a chunk that will be rendered? This would also match the `RenderedModule` type nicely.

Reply:
Sure.